File: proyecto6/libro.php
<?php

class Libro {
    public $titulo;
    public $autor;
    public $paginas;

    public function mostrarInfo() {
        echo "El libro ".$this->titulo. ", cuyo autor es ".$this->autor.", tiene un total de ".$this->paginas." páginas";
    }
}

$libro = new Libro();
$libro -> titulo = 'xxx';
$libro -> autor = 'xxx';
$libro -> paginas = 23;
$libro -> mostrarInfo();

?>

File: Java5/Ejercicio_55/miLibro/Libro.java
package miLibro;

public class Libro {
	
	String titulo;
    Autor autor;

    public Libro(String titulo, Autor autor) {
        this.titulo = titulo;
        this.autor = autor;
    }

    public void mostrarDatos() {
        System.out.println("Título: " + titulo);
        autor.mostrarDatos();
    }
}
